recipes/sigmut/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "sigmut" %}
+{% set version = "v1.0" %}
+{% set sha256 = "d0060805ffb819461ad5666be53f11879ba0ec37a79496bda6bb1af32f8c02b8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/lgpdv/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - matplotlib >=3.1.0
```

Comment:
These should probably be matplotlib-base

Reply:
Thanks a lot for your suggestion. I will modified the file and try the PR again.